Examples/LambdaFunctions/README.md

```diff
@@ -85,3 +85,86 @@ The SAM template will provide an output labelled `LambdaApiGatewayEndpoint` whic
 ***Warning:*** This SAM template is only intended as a sample and creates a publicly accessible HTTP endpoint.
 
 For all other samples use the AWS Lambda console.
+
+### Deployment instructions using AWS Serverless Framework
```

Comment:
`AWS Serverless Framework` sounds like "serverless" is a official tool provided by AWS. But that would be SAM.

I recommend changing this to something else that makes it clear that this is about serverless.com

Reply:
ee36b6d